apps/ledger-live-mobile/src/screens/PostOnboarding/PostOnboardingHub.tsx

```diff
@@ -0,0 +1,244 @@
+import React, { useCallback, useEffect, useRef, useState } from "react";
+import {
+  Divider,
+  Flex,
+  Icons,
+  Log,
+  Notification,
+  Text,
+} from "@ledgerhq/native-ui";
+import { useTranslation } from "react-i18next";
+import { ScrollView, StyleSheet } from "react-native";
+import { SafeAreaView } from "react-native-safe-area-context";
+import styled from "styled-components/native";
+import { useFocusEffect } from "@react-navigation/native";
+import Animated, {
+  interpolate,
+  runOnJS,
+  useAnimatedStyle,
+  useSharedValue,
+  withDelay,
+  withTiming,
+} from "react-native-reanimated";
+import { StackScreenProps } from "@react-navigation/stack";
+import {
+  useAllPostOnboardingActionsCompleted,
+  usePostOnboardingHubState,
+} from "@ledgerhq/live-common/postOnboarding/hooks/index";
+import { clearPostOnboardingLastActionCompleted } from "@ledgerhq/live-common/postOnboarding/actions";
+import { useDispatch } from "react-redux";
+import { Fade } from "@ledgerhq/native-ui/components/transitions";
+import PostOnboardingActionRow from "../../components/PostOnboarding/PostOnboardingActionRow";
+import { NavigatorName } from "../../const";
+
+const SafeContainer = styled(SafeAreaView).attrs({
+  edges: ["left", "bottom", "right"],
+})`
+  flex: 1;
+`;
+
+const AnimatedFlex = Animated.createAnimatedComponent(Flex);
+
+type Props = Record<string, never>;
+
+const PostOnboardingHub: React.FC<StackScreenProps<Props>> = ({
+  navigation,
+}) => {
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+  const { lastActionCompleted, actionsState } = usePostOnboardingHubState();
+  const [popupOpened, setPopupOpened] = useState(true);
+  const { actionCompletedHubTitle, actionCompletedPopupLabel } =
+    lastActionCompleted ?? {};
+
+  const clearLastActionCompleted = useCallback(() => {
+    dispatch(clearPostOnboardingLastActionCompleted());
+  }, [dispatch]);
+
+  useEffect(
+    /**
+     * The last action context (specific title & popup) should only be visible
+     * the 1st time the hub is navigated to after that action was completed.
+     * So here we clear the last action completed.
+     * */
+    () => clearLastActionCompleted,
```

Comment:
shouldn't this be a call to "clearLastActionCompleted"? like this it's only returning the function but not calling it

Reply:
here we return `clearLastActionCompleted ` which will be called as the cleanup function of that `useEffect`
I made this change in reaction to https://github.com/LedgerHQ/ledger-live/pull/943#discussion_r949052825

Reply:
Ah, I see, didn't realize it was on purpose.